Test/CoreSDK.Test/Shared/DataContracts/PerformanceCounterTelemetryTest.cs

```diff
@@ -2,8 +2,9 @@
 {
     using Microsoft.ApplicationInsights.Channel;
     using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using System;
```

Comment:
OK, I think I know why this was not flagged--StyleCop is not running for test code. As I said, will fix